### PR TITLE
Makes Research Director into a protected job and adds Loyalty Implant

### PR DESCRIPTION
**Pull request by Lady-Luck**
As you know, Science roles make for incredibly strong antagonists. However, with the Research Director, that's taken to a whole 'nother level. It's like if the Head of Security could be a traitor - the amount of power you have is insane.

If you're an antagonist that utilizes conversion mechanics, well, you've pretty much won the game. Just convert your department and go mad with power.

By turning Research Director into a protected job, it actually reduces the power of Science antagonists as a whole. The RD can be more readily trusted by Security when it comes to investigations, and the RD can no longer be converted.

I believe this also has the potential to move the RD away from being seen as a Super Scientist and as more of a... well, Research Director.

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Yeah, no.

**Comment by Kevak** [comment]
Ew no.

**Comment by Lady-Luck** [comment]
I'd appreciate reasoning. I've given plenty, and have received none.

**Comment by Fox-McCloud** [comment]
Failed Travis.

You have a defined variable that you don't do anything with.

**Comment by Lady-Luck** [comment]
?RD traitor isn't horrendously more powerful than your average scientist--he has some additional equipment (baton, vest, and laserpointer), and more access, but other than that, the vast majority of things he can do, other scientists can do and be just as strong.

It's not strictly about their power as an individual, it's about their power as a department head. You know, talking to people. They're able to justify just about anything due to their role and make orders that befits their goals. By making the RD into a trustworthy overseer, the chance of that department being swiftly taken over falls significantly.

**Comment by Spacemanspark** [comment]
Generally every time I see a traitor RD it ends in headache and murder everywhere, I've seen quite a few people (at least on the discord) talking about wanting RDs to not be an antag.

**Comment by counterfeitguise** [comment]
For the love of god, please merge this. It's not an "i ded plz nerf" anymore, it's a legitimate _problem_. RDs and Scientists in general wipe the floor with the station for a ridiculous number of reasons - RDs actually doing their jobs - Directing Research, like they're **hired to do** - will make science antags actually have to _work_ for their murderbone sprees.

**Comment by Fox-McCloud** [comment]
It's also going to turn RD's into the biggest retarded validhunters around. Now they have protection against everything *and* access to tons of guns and mechs (that no HoS or Captain is going to care about anymore because they know he can never be antag now).

Can't convert them, can't ever be antag---HoS/Cap might as well give them all access and guns at that point, because it's 100% safe to do so, because it's hardcoded trustworthiness.

**Comment by Lady-Luck** [comment]
I don't understand why you think we're going to see any more RD validhunters than we have right now. It's just eliminating the possibility of being a traitor. And also, we don't allow the HoS or Captain to be walking armories, why do you think RD is going to be treated any differently in that regard?

**Comment by Lady-Luck** [comment]
Also, the assumption that RDs will be considered by anyone to be 100% safe to do whatever they want just because we know they're not antagonists is laughable - there's still a human being controlling that character, and god only knows what 'brilliant plan' they've come up with.

**Comment by Davdichan** [comment]
Yes please. Given the amount of hate science gets and the sheer number of PRs flung against science to nerf it more and take things away from them, this is probably one 'nerf' I can get behind.

As for @Fox-McCloud I don't see this turning into the HoS/Cap giving him access to guns. He's still locked out of the boxes and giving the RD access to unlock them is just going to make the RD even bigger of a target, similar to the HoP.

RD has near unrestricted AI access, Tcomms, Toxins, Robotics, Xenobio, Genetics, Tech Storage and Basic Sec. He lacks basic medical but thanks to cloning he can walk right into medical anyways. So as a traitor he's got basically free access to Chems and Medicine, can free sabotage cloning, can bomb sec at will and can easily rig up telecoms to short out on command from his PDA.

**Comment by theColdflame** [comment]
:+1: 
This needs to happen. The points regarding validhunting and meta-trusting are somewhat valid, but the bad hugely outweighs the good here. You don't see security, IAA, the NT rep, or captain being given all-access and guns to hunt with. As it stands, the opposite problem is present- The current expectation is that the RD is an antag, especially given the habit certain RD players have of semi-regularly  going to cryo when not an antag.

RD antags are just straight-up overpowered, no amount of changes to science will fix this problem. They have full access to science and the expectation they will do nothing but make bombs/xenobioarmies/guns/deathchems/lube/tcomms script with a mute function/telesci/combat mechs/whatever, along with access to:

-Genetics: Even alone, genetics is a very strong asset to an antagonist. Given around ~40 minutes of time investment (potentially even less) the RD can have all of the combat powers, which in combination with the rest of the resources afforded to them make them pretty much unstoppable.
-AI upload: The RD can just walk in, swipe a traitor board, and walk out when nobody's looking. And they do.
-Secure tech storage: Was walking into upload too much effort? Just go pick up an upload board!
-Basic department access- Not too major, but in combination with their huge amount of resources they also have the ability to waltz into any department they want without the risk involved in breaking in
-Teleporter: Free voidsuit and handtele. Have fun stopping a hulk with the hand teleporter.

Conversion antags in science also have far too easy of a time, given they're one of two departments which lacks regular travel from people outside the department and loyalty implanted personnel. The RD would create an obstacle to conversion antags opting to convert all of science; The scicult strategy would still be viable, but it wouldn't be the completely obvious choice. 

This would also the first step towards RD being something other than scientist+,  and we might actually see RDs doing their job if this happens.

**Comment by Lady-Luck** [comment]
@theColdflame Don't forget telecomms.

**Comment by monster860** [comment]
If this does go through, we'll need to figure out the telegun.

Let's make it a very expensive item available to all traitors!

**Comment by Davdichan** [comment]
Like all guns it'd be available with the purchase of an emag and RnD doing their job. Lock boxes are just cancer like that.

**Comment by monster860** [comment]
Are you saying make the telegun available to RnD?

That sounds like a horrible idea. Imagine security getting the power to teleport people to the brig instantly.

**Comment by Aurorablade** [comment]
I can't agree with this. I might consider it if the same were done to the cmo,ce and hop but even then not totally agree

**Comment by pipextiny1** [comment]
So I just read this and basically every one who wants RD antag removed gave a valid point.
On the other hand people who still want it in the game was like "no! they will be valid hunters a lot like they are right now when they don't roll it.
I want this RD antag trash out of the game

**Comment by FreeStylaLT** [comment]
I believe I am against this change. 

@theColdflame 

RD is as overpowered as any other head that is made traitor, especially HoP. Yes, RD gets access to all of the things listed by default as his job, but that is the whole point. Science is not meant to be a reliable, stable part of the station which always has someone you can all metaknow to not be an antag. 

The whole point of the amount of tools and toys science gets is that it needs to be tamed, to be managed, to be looked after, but not by the RD alone, because even the man himself can be turned into a whacked-off antag going on a spree.

This is by design. Lately the crew has been able to co-operate very efficiently against antagonists. We get genetics infusing sec every other, if not every round, even setting them on fire if the round is shadowlings, we have coordinate efforts with the chaplain during cult rounds, on top of that science can aid in the hunt of any of them with their chemical concoctions. 

Why reduce the shrunken amount of entropy in the server by making someone with access to so much to cause damage, be effectively _never_ able to utilize it? What purpose does it serve other than making the rounds more dull and long as the threats are even more reliably dispatched?

Yes, you can argue the logic of CC making sure that the RD they're sending in isn't a whackjob, but senseful thinking has been thrown out when antags were able to be identified due to looking "Uncannily repulsive". Metaknowledge is part of the server and its players' playstyles at this point. We must make decisions with it in mind.

**Comment by Earthdivine** [comment]
If this goes through you would need to make that change to all the heads, because if not then it's very much a reflexive change and those never go well in the long term. You see the RD and HoS implanted but not the CE, CMO, or HOP. That doesn't make sense from a balance point or from a lore point, and it just looks lazy by design, and will confuse new players as well.

**Comment by monster860** [comment]
The HoP is implanted though

**Comment by FreeStylaLT** [comment]
@monster860 No, he isn't. The implant was removed from HoP a whiiile ago, maybe a year or more ago.

**Comment by Lady-Luck** [comment]
> If this goes through you would need to make that change to all the heads, because if not then it's very much a reflexive change and those never go well in the long term. You see the RD and HoS implanted but not the CE, CMO, or HOP. That doesn't make sense from a balance point or from a lore point, and it just looks lazy by design, and will confuse new players as well.

This is not reflexive. This is something that's been talked about for weeks on the Discord. The reason CMO, HOP, or CE isn't being targeted is they don't have nearly the power an RD does as an antagonist. They get caught all the time.

Also your idea of what makes good lore is subjective. From my perspective, it makes more sense for RD to have rigorous hiring practices just like Captain and HoS, if not moreso, because NanoTrasen is a science corporation and having a traitor RD is going to lose them a shitload of money.

**Comment by Lady-Luck** [comment]
>RD is as overpowered as any other head that is made traitor, especially HoP. Yes, RD gets access to all of the things listed by default as his job, but that is the whole point. Science is not meant to be a reliable, stable part of the station which always has someone you can all metaknow to not be an antag.

Arguing about what is 'meant to be' doesn't seem productive at all. And making RD pure loyalist isn't suddenly going to make Science a completely stable department. Scientist antags will still be insanely strong.

>The whole point of the amount of tools and toys science gets is that it needs to be tamed, to be managed, to be looked after, but not by the RD alone, because even the man himself can be turned into a whacked-off antag going on a spree.

Both Robotics and R&D have privacy shutters, and even if you have access to Science, you probably don't have access to their computers. They have more space than any other department and are thus spread across a large area that needs to be supervised. And if the department is compromised, the central hall lends well to lynching any unsuspecting staff member trying to 'tame' them because they're surrounded by every single sub department and they might even have the AI on their side.

Like I said, it's nuts to think that the RD is suddenly going to put an end to all science antagonists, because Scientists walking around with insanely powerful things is seen as the Norm that no one has to write off on, other than warmechs.

**Comment by TullyBurnalot** [comment]
As much as I know that Traitor RDs are probably one of the worst Traitors to go up against, that's not because they're the RD. It's because they have access to Science.

Any regular Scientist can quite easily arm themselves just as much as the RD. The only difference between a Scientist and the RD is Bridge/Upload/Telecomms access. The first one doesn't matter, you can recreate Upload computers for laughably low price, and unless you happen to be a really competent/crafty Traitor, you won't be using Telecomms. And if you **are** a crafty antagonist, you can quite easily get to Telecomms with your basic equipment, no Syndicate gear required.

That said, I disagree with the notion that patching this in would suddenly turn all RDs into supreme validhunters that will receive the full blessing of Sec and Command. That is a ridiculous notion. Player mentality does not do 180s like that.

The issue here is not that the RD is powerful, but that Science can quite easily produce gear that far outperforms Syndicate equipment in offensive terms, not to mention the fact that they can **mass produce** some of the best implants in the entire game, for a ridiculously low price, assuming Mining is half competent (and they are, 99% of the time). This very much feels like a knee-jerk "Nerf 'dem RDs" PR. And I should know, I received my share of the flak writing Science SOP precisely because of that.

Ultimately, though, I personally believe that if you want to loyalty implant any Head of Staff, do it to the Head of Personnel. They have instant, roundstart access to Full Access IDs and are usually not scrutinized in the slightest. The RD, by contrast, has Science+Command+Telecomms access and is almost always looked at with some measure of suspicion, especially if anyone even so much as catches a glimpse of something that looks like a lockbox.

Loyalty implanting the RD does not seem necessary. Limiting Science access to ridiculous amounts of highly powerful guns, maybe? But that's a discussion for another day.

TL:DR: It's not the rank, it's the Department. No implant required.

**Comment by theColdflame** [comment]
This is not a kneejerk or reactionary PR- a lot of players have been vocal about this for a long time.

@FreeStylaLT 
The statement that other heads are of equal power to the RD when antags is, well, wrong. Comparing RD antag rounds to those with HoP/CE antag rounds make a huge difference clear. Heads are strong antags but by no means overpowered, whereas the RD certainly is. The argument can be made that the HoP is stronger from a mechanical standpoint, but they're under far more scrutiny and aren't expected to be making bombs/genetic powers/xenobio armies anyway.

Your opposition to this PR seems to be because you disagree with a lack of chaos during rounds in general, which is entirely outside of the scope of this PR. RD antags have absolutely zero to do with the issues you raised in your post.

The reason people have been so vocal about removing RD antags is the fact that they are overpowered to the point of detracting from the enjoyment of the game when they are involved- you cannot defend yourself against an RD antag unless you're prepared to lower yourself to the same level of powergaming they're notorious for.

**Comment by Lady-Luck** [comment]
@TullyBurnalot I think you're missing the point. The idea is that this puts someone in the Science department that can be trusted. Right now Security has to conduct its own investigations into something they're not expected to know anything about because the RD cannot be trusted. It's one of the hardest departments to get into and requires a lot of babysitting from the AI to open the obscene amount of doors.

Science is just too much of a safe haven for antagonists and making the RD into someone that is always against them and is considered a reliable source for investigations will help bring down the power of science antags an indirect, but very effective manner.

**Comment by TullyBurnalot** [comment]
@theColdflame Technically speaking, the HoP is in a better position to abuse Science than the RD is. The RD needs to buy an Emag to open lockboxes, whereas the HoP can just give themselves full access to anything Armory-level.

@Lady-Luck You can technically extend that reasoning to any Department, though I understand where it is you're coming from. I still maintain that the main issue is the power creep inherent to RnD, especially when Mining is competent and well-staffed. However, forcing the RD to be always-loyalist seems like a "Obvious Rule Patch"-type solution. Perhaps a better, more interesting, solution would be to change around the inner workings of RnD. Though, of course, I am fully aware that's a magnanimous hornet's nest that no one would ever want to touch.

Frankly, my opinion is thus: if you're going to implant the RD as a "we need someone to trust in that Department" precaution, you may as well implant all the other Heads as well. Otherwise it just feels artificial.

**Comment by Lady-Luck** [comment]
>Frankly, my opinion is thus: if you're going to implant the RD as a "we need someone to trust in that Department" precaution, you may as well implant all the other Heads as well. Otherwise it just feels artificial.

I disagree. This is a RESEARCH Station, and NT has always done things on a 'need to be done' basis. HoS, Captain, obviously needs to be done. And the guy that has more influence on their profit than anyone else on the station? You should probably make doubly sure he's not a Syndicate agent that's going to steal all your research. It makes perfect sense.

**Comment by theColdflame** [comment]
@TullyBurnalot Walk into toxins mixing as the HoP with an all-access ID and make a bomb, see how many steps you make outside before sec show up. 

Game mechanics are not the only factor with regards to the strength of a job as an antagonist.

**Comment by TullyBurnalot** [comment]
@Lady-Luck You also have the "Person in charge people do not die/get cloned" and "Person in charge the station has power and the engine does not murderlize everyone".

Other Heads of Staff also have very important IC roles that you would technically want to ensure were done correctly and were not sabotaged.

@theColdflame I can agree. That said, a good HoP Traitor can do a hell of a lot more damage than a good RD Traitor, and yet, they are to remain non-implanted. From an IC perspective, they are also the second in command of the Cyberiad, and more often than not fill the role of Captainship. And yet, unimplanted.

**Comment by Lady-Luck** [comment]
Also, even though the HoP is quite powerful, they also perform a very busy service job. It can be difficult for them to find time to follow their antagonist objectives, especially when the SOP binds them into getting to their desk in a reasonable amount of time. They have to interact with a lot of people and they're in an environment where it's easy to check up on them.

The fact they're second in command is almost more of a bane than a boon to a HoP antagonist unless they were lucky enough to roll objectives that involve stealing the Captain's equipment. You will be kept incredibly busy and you will be eternally thankful when a Captain shows up.

And I wholy disagree with the notion that the HoP is more capable than the RD of damaging the station. I have never once seen that to be the case. People are automatically suspicious of them whenever they're outside of the Bridge. He doesn't even have full social influence of cargo - you can bet your ass the anal retentive Quartermasters are going to ask the Captain if the HoP is supposed to be ordering guns.

**Comment by Spacemanspark** [comment]
@TullyBurnalot Thing is, though, if the HoP is out of their post for more than ten minutes they generally get yelled at (As it should be.), whereas the RD can just pretty much do whatever.

**Comment by Lady-Luck** [comment]
I've seen more HoP terminations than any other head of staff.

**Comment by FreeStylaLT** [comment]
The examples I spoke of were not necessarily issues, just statements of facts, which were that the crew has been co-operating more extensively against antags lately, causing more rounds to end unsuccessfully on the antag side, leading to more dull and long rounds. RD being neutered in antagonistic potential would only make that issue _worse._ 

Regardless of how much potential other scientists retain, it makes one scientist with access to everything reliable and trustworthy, which would inevitably lead to him working against antags utilizing all of the same assets, not every round, but enough to feel even more unfair to the antags. 

IC assumptions regarding heads are insubstantial, as the same can be argued regarding every head, "They should be trusted to do their job appropriately", but just the mere nature of science is the fact that is not mean to be reliable, by design. Bringing up "meant to" here is very relevant, as it is functioning right now as it was meant to.

**Comment by Lady-Luck** [comment]
>The examples I spoke of were not necessarily issues, just statements of facts, which were that the crew has been co-operating more extensively against antags lately, causing more rounds to end unsuccessfully on the antag side, leading to more dull and long rounds. RD being neutered in antagonistic potential would only make that issue worse.

Don't you think it would be easier to nerf the power of Science antags and then buff traitors as a whole rather than try to bring up every single department to Science's level?

**Comment by TullyBurnalot** [comment]
The issue with nerfing the capabilities of Science Traitors is that the inner working of Science are a hornet's nest that no one wants to get their hands on. Considering how long it's been a part of the game, and how even the tiniest of changes to the balance of it bring the Science regulars out of the woodworks crying for blood, changing Science so that it does not provide as much of a power level to its Traitor members is a **fuckass huge** endeavor.

**Comment by theColdflame** [comment]
@FreeStylaLT 
That has nothing to do with this PR.

RDs are completely disproportionate to other antagonist jobs in strength; that's a major balance issue and should be resolved, regardless of how strong/weak antags are as a whole. Buffing antags in general should be discussed elsewhere.

**Comment by FreeStylaLT** [comment]
Yes, they are disproportionate, it is a consequence of what Science is and what it has access to. Thus consequently the RD can be more dangerous than other heads. But the main issue is that as I have said twice now, the state of how the community behaves cannot take another addition of neutering antagonistic behavior. 

Yes, this may seem like a separate thing altogether, but it is a relevant and big enough one to warrant not getting this change through.

I will thoroughly oppose any possible implementation like this that does nothing but interfere with antagonists or neuters their potential, no matter how seemingly destructive or powerful it may be at this current moment.

**Comment by Lady-Luck** [comment]
I think your philosophy is flawed. If you don't make things proportional, then it's hard to do generalized changes, because then people will complain that the disproportional antag roles will become too powerful.

It's a catch-22.

**Comment by FreeStylaLT** [comment]
I do not disagree with your statement, but I do not believe that implanting the RD is the right way to tone down science antagonism, which is not even entirely necessary to do right now, due to the reasons I've already stated.

**Comment by theColdflame** [comment]
Traitors have been nerfed twice over the past week due to the actions of RD antags. Said nerfs were probably justified regardless, but it can't be argued that RD antag is some bastion of a much more dynamic, more interesting gameplay.

RD antag is boring and unfair. That's a fact. Removing them will have a positive impact on round quality and will not 'worsen' the community (I disagree with your opinion on the state of the server but as I've already said that's out of scope here.)

**Comment by Lady-Luck** [comment]
Your reasoning is that sci antagonists aren't overpowered because everyone has been working together. According to that logic, means that almost every other type of antagonist is underpowered because they don't have the abilities Science has.

Because Science is such an outlier, it makes far more sense to bring down the power of antagonism in that department, and then make a sweeping change that all traitors can benefit from.

**Comment by FreeStylaLT** [comment]
My reasoning is that the overpowered nature of the entire department as antagonists should not be toned down further from this current point, at the very least not in the way as proposed in this PR. 

Yes, Science is an outlier in power, but as Tully said, we cannot simply rebalance it _again_ and make more reworks or huge sweeping changes to how the department's four completely separate systems function in order to tone down antag potential because it does not match up with other departments. In fact, that does not seem like a good idea, given how science has always been an outlier in that direction. Are the tools they have disproportionate to other antags that may appear in other departments? Maybe. Are the tools they have disproportionate to what the crew can get to fight them? No.

Yes, everyone has been working together. This used to be an uncommon, but satisfying occurrence,  but has become rather commonplace with the addition of more competent, mechanically-driven players. I do not wish to change that, but I must take that into account when deciding if nerfing antagonists, science or not, should be warranted. To me, it is not. Not right now, not like this.

**Comment by theColdflame** [comment]
You're looking entirely at the bigger picture whilst completely ignoring the smaller one.

RD antag sucks. That's the best word for it. Nobody is going to have fun when the RD powergames and massacres people.

**Comment by Lady-Luck** [comment]
>Yes, Science is an outlier in power, but as Tully said, we cannot simply rebalance it again and make more reworks or huge sweeping changes to how the department's four completely separate systems function in order to tone down antag potential because it does not match up with other departments. In fact, that does not seem like a good idea, given how science has always been an outlier in that direction. Are the tools they have disproportionate to other antags that may appear in other departments? Maybe. Are the tools they have disproportionate to what the crew can get to fight them? No.

In not bringing down the power of Science antagonists, this significantly hinders every other department from receiving the antagonist buffs they need. You'd need to make changes one job at a time, and possibly redesign entire departments to give them greater potential.

Never in the history of game design has 'Instead of nerfing x, let's buff everything else,' ever worked out. This mindset has a legacy of stalling proper balance.

This is a Science antagonism nerf that actually DOESN'T take away any of their toys for once and doesn't require an insanely massive overhaul that no one wants to do.

Remember, the HoP used to be loyalty implanted and is not anymore because it was no longer needed. If this goes through, it does not have to be permanent. If Science ever does get that sweeping overhaul you're hoping for, this can be undone just like HoP loyalty was.

**Comment by Draconic-saint** [comment]
Here's what I think.

Should the RD( and by extension the other heads of staff) be loyalty implanted on round-start?

Nope. They don't require the protection in my eyes, simple and done. ICly they have the blueshield, the security forces and the majority of their departement to back them up. From a design standpoint, it'd make them another giant obstacle for certain gamemodes to be overcome, which they don't need. Because we're all talking about fixing traitor, but this has ramifactions for other gamemodes aswell. Shadowling and cult in particular would have to deal with ANOTHER obstacle that's damn hard to overcome. So in interest of not upsetting that balance too much, i'm against implanting the RD.

RD traitors going hogwild with all of science's achievements backing them up, can indeed be an issue. But so can scientists that are good at their job. Sure RD's have that added acces, that level of credibility, that presence of being well, the good guy. And if he's a good traitor, he can and will use that to his advantage. 

But isn't that the point of being a good traitor, to use EVERY tool at your disposal? I once read somewhere :

>  talking fast and well is a traitors best weapon.

While I personally prefer the SAW, I can see the value in that quote. And that level of suspicion, that feeling that _everyone_ can be a bad guy, is a reason why I like playing ss13. I started playing in an age when EVERYONE could be traitor. Captain, HoS, detective...

Hell, the most vivid traitor memory I have is how I (ab)used my detective authority to lure my target out to a "reconstruction of a shooting". One with live ammo. So i'm pro having untrustworthy people you technically **should** trust.

To summarise, yes GOOD traitor RD's are in a spot of too much power, but only marginally more than a GOOD traitor scientist. How to fix this? I have no clue, and i'll muse on it, but I don't like removing the RD from the traitor list, and I'm straight up opposed to implanting them.

**Comment by TheDZD** [comment]
Science as a whole is a stupidly powerful department for antagonists, it just has too many things that are incredibly useful, but almost entirely to antagonists (deathchem mixes, simple_animal/golem, toxins, etc). 

That said, I'm not quite so certain if implanting the RD is going to fix that.

**Comment by Spacemanspark** [comment]
I'm fine with just de-traitoring the RD, don't really care too much about the implanting.

**Comment by Kyep** [comment]
Voted no for several reasons:

1) RD is not uniquely deserving of a loyalty implant. HoP is actually in a stronger position, because they can easily all-access themselves as a traitor and take anything from anywhere. Not to mention, they start with armor and an egun. Not only are they NOT loyalty implanted, but they were in the past and it was removed. I don't see a good reason to do the exact opposite to RD as was done to HoP.

2) Antags are already pretty tame on Paradise. I would argue, too tame. If people who play antag RDs seem to be so much stronger than other antags, perhaps that's because (A) more experienced players tend to play Science, and (B) most antags are pretty weak. If you want to reduce a power discrepancy, buff other antags rather than nerfing the RD.

3) Most of the times I see an RD antag, they're not using their department to further their antaggy goals by giving orders to subordinates. They're usually operating mostly alone. 

4) With loyalty implants, RDs become more validhuntey, which is very, very bad. I play various command jobs a lot, and I can tell you loyalty implant = given a lot of leeway by other crew.

5) There is a general campaign to nerf Science, which I consider totally misguided. Much of the nerfs to Science focus on removing the potential for fun, because people are jealous that Science is more fun to play than most other departments. IMHO the solution is to make other departments MORE fun, not making Science less fun. I will say that twice, since it is so important: instead of campaigning to remove every bit of fun in Science, start thinking about what could be done to ADD fun to other departments. People keep advocating for "one more nerf", but it never seems to be enough, and the end result is just a net loss of fun for the game.

6) The problem of "heads cryoing if not antag at roundstart" can be addressed in other ways. Also it applies to many jobs other than RD, so I don't see why RD should be singled out because of it.

7) I am very skeptical about any change suggested to counter "powergaming". It seems to me that "powergaming" is a charge levied against people who play the game "too well". Nerfing the strategies they use won't stop them - their nature as players just means they will find another strategy. 

8) This is going to have side-effects. For example, hindering cult rounds. Cult often grows in science and having it overseen by an implanted person with probable access to weapons (due to more trust) is going to affect more people than just the RD. We already have a problem with to many long/boring rounds due to antags never getting a foothold. I don't like the idea of making that problem worse.

**Comment by Da-Dman234** [comment]
Right, so browsing through this I'm against it. I'll go ahead and throw in that I only ever play scientist (but never RD). But this PR is addressing absolutely nothing. It's piss easy to achieve the power levels you guys are talking about as a normal scientist (If you know what you're doing). Want genetic powers? Use unstable muta and Penetic/Mutadone. Want an AI upload? Print out the boards and use them. Want a big stompy mech to demolish the station? Build a exosuit fab just like everyone does every single god damn round. The only reason you very rarely see regular science players go off the hook because objectives never call for it.

All implanting the RD will do is kill the amount of players that take up the position, because why bother doing so if you can just be a scientist with none of the responsibilities, none of the implants, and just as much power?

If you have a problem with science find a different way to nerf it or buff other departments, this is definitely not the way to do it.

**Comment by Ar3nn** [comment]
I'll just add to this that you need to take off the rose-tinted glasses that you think the game needs to be easy for either side. I've advocated against nerfs to anything related to antags in the recent past because this game feels entirely too much like a sandbox game at this point, with an annoyance on the side. I throw around the term "hugbox" a lot lately, and that's because other people started using it, because it definitely has started applying to this codebase. I took up the challenge of deciphering this code language to look into some ways of potentially buffing things back to levels of fun and interesting play that existed before LINDA and all the other crap that's been thrown in since then. The fixes and things that have been added are welcome, but a vast majority of things I've seen happen since LINDA have been in the direction of making the regular play a boring spreadsheet of do this for x time, proceed to next step on list J, and antags have no place here. I'm not saying hand antags their objective on a platter by making them overpowered, but they are in a MUCH WEAKER state than they were when I joined up, and it seems rather unjustified. A lot of other things have been removed/never added because, while fun, they grant GREYTIDE access to things that they need to grief the station, and while that doesn't apply here, its another thing I'm 100% against. Nerfing or removing or denying features over something that is NOT an IC concern, doesn't happen every round, and is fixed anyway by banning the person responsible and waiting for the next restart should not be considered reasonable. I'll drop that there, and just say that as a long-time player of this server, I do not want this server to kill itself off in the interest of making it "safe" or "easy" which will happen if this stuff keeps getting merged.

**Comment by Faryaa** [comment]
I must point out that we still need to do something with RD. because for now playing traitor RD have almost no difference from playing playing loyalist RD. Both will be a bit crazy weird dudes doing science for fun, injecting all of the genes, building combat mechs and drive them around, e.t.c. The only actual difference is a reason why RD do that - for harmless amusement or for evulz.
While any other traitor head of staff have notably different behavior from banevolent head of staff.
CE turn into sabotage agent, HoP is goin' Batman and do traitorous things with fake name and all-access, CMO become typical Herr. Doktor who throw the Hippocratic Oath away.

**Comment by Lady-Luck** [comment]
While there's currently a majority support for this PR, unfortunately very little administration backs it, so it's probably not going through. I'd like to explore alternatives that can be realistically done sooner rather than later.

**Comment by Ar3nn** [comment]
It looks about evenly split to me, lots of people on either side. If you want alternatives, I'd be glad to help you hash some out. For example, add more fun features to other areas of the station that could be utilized by traitors or maybe even the other antags to do evil? Add new fun tator uplink items that are joblocked? stuff like that.

**Comment by Lady-Luck** [comment]
RIP. Despite the support, almost all admins are against it. It ain't happening.

**Comment by Deanthelis** [comment]
This, this shit is why I don't get on the github much anymore. Tater RD is a time-honored tradition of scientific fuckery. Hell, I was against protected Security Officers - what, corrupt cops just don't exist anymore? - and this is just silly.

**Comment by Lady-Luck** [comment]
Dude. The commit is closed. Move on.

**Comment by Lady-Luck** [comment]
@Deanthelis Dude. The commit is closed. Move on.

**Comment by SigholtStarsong** [comment]
This shit needs to happen.

**Comment by Ar3nn** [comment]
Make your own PR if you really want it to happen, get it past the maintainers who weren't jazzed about it to begin with. Don't revive dead and closed PRs.
